definitions/procedures/content/prepare.rb

```diff
@@ -6,7 +6,7 @@ class Prepare < ForemanMaintain::Procedure
 
       confine do
         # FIXME: remove this condition on next downstream upgrade scenario
-        !feature(:instance).downstream
+        !feature(:satellite) || feature(:satellite).at_least_version?('6.9')
```

Comment:
Here, `downstream` refers to  feature `satellite || capsule` depending on which one is available. 
Just for the sake of curiosity, what is expected behavior for capsule? as this procedure is currently hidden on both Satellite and Capsule.

Reply:
The starting and stopping of pulpcore services only applies to satellite, not capsules.